recipe_sha: 2820548785723f9a229191965eb47a0688edc023037eb15abadf5b6d3f5a9702  # The commit at which this recipe failed to build.
skiplist: false # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-
      perl:          5.32.1-4_h31becfc_perl5 conda-forge
      readline:      8.2-h8fc344f_1          conda-forge
      zlib:          1.2.13-h31becfc_5       conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
  [34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-aarch64                  134.0 B @   3.6MB/s  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   6.3MB/s  0.0s
  [?25h[34mReloading output folder: [0m[34m/opt/conda/[0m[34mconda-bld[0m
  [?25l[2K[0G[] 0.0s
  [2K[1A[2K[0Gopt/conda/conda-bld/linux-aarch64                  134.0 B @   3.2MB/s  0.0s
  opt/conda/conda-bld/noarch                         127.0 B @   5.8MB/s  0.0s
  [?25hMamba failed to solve:
   - libzlib >=1.2.13,<1.3.0a0
   - bzip2 >=1.0.8,<2.0a0
   - wget
   - perl >=5.32.1,<5.33.0a0 *_perl5
   - aria2
   - zlib
   - gawk
   - libstdcxx-ng >=12
   - _openmp_mutex >=4.5
   - libgcc-ng >=12

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested aria2
  -

  Leaving build/test directories:
    Work:
   /opt/conda/conda-bld/work
    Test:
   /opt/conda/conda-bld/test_tmp
  Leaving build/test environments:
    Test:
  source activate  /opt/conda/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl
    Build:
  source activate  /opt/conda/conda-bld/_build_env


  Traceback (most recent call last):
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 141, in mamba_get_install_actions
      solution = solver.solve_for_action(_specs, prefix)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 244, in solve_for_action
      t = self.solve(specs)
    File "/opt/conda/lib/python3.8/site-packages/boa/core/solver.py", line 234, in solve
      raise RuntimeError("Solver could not find solution."  error_string)
  RuntimeError: Solver could not find solution.Mamba failed to solve:
   - libzlib >=1.2.13,<1.3.0a0
   - bzip2 >=1.0.8,<2.0a0
   - wget
   - perl >=5.32.1,<5.33.0a0 *_perl5
   - aria2
   - zlib
   - gawk
   - libstdcxx-ng >=12
   - _openmp_mutex >=4.5
   - libgcc-ng >=12

  with channels:
   - conda-forge
   - bioconda
   - defaults

  The reported errors are:
  - Encountered problems while solving:
  -   - nothing provides requested aria2
  -

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/conda/bin/conda-mambabuild", line 10, in <module>
      sys.exit(main())
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 256, in main
      call_conda_build(action, config)
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 228, in call_conda_build
      result = api.build(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/api.py", line 180, in build
      return build_tree(
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 3078, in build_tree
      packages_from_this = build(metadata, stats,
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 2115, in build
      create_build_envs(top_level_pkg, notest)
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1993, in create_build_envs
      raise e
    File "/opt/conda/lib/python3.8/site-packages/conda_build/build.py", line 1972, in create_build_envs
      environ.get_install_actions(m.config.test_prefix,
    File "/opt/conda/lib/python3.8/site-packages/boa/cli/mambabuild.py", line 150, in mamba_get_install_actions
      raise err
  conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-aarch64: {MatchSpec("aria2")}
# Last 100 lines of the build log.
